roles/ceph-defaults/defaults/main.yml

```diff
@@ -288,6 +288,9 @@ generate_fsid: true
 
 ceph_conf_key_directory: /etc/ceph
 
+# Permissions for keyring files in /etc/ceph
+keyring_permissions: '0640'
```

Comment:
Do you mind prefixing with `ceph_`? Thanks.

Reply:
done.